usaspending_api/api_contracts/contracts/v2/awards/award_id.md

```diff
@@ -148,15 +148,26 @@ This endpoint returns a list of data that is associated with the award profile p
 + `date_signed`: `2006-01-15` (required, string, nullable)
 + `subaward_count`: 430 (required, number)
 + `total_subaward_amount`: 35345353453 (required, number, nullable)
-+ `cfda_objectives`: `Some HTML string` (required, string, nullable)
-+ `cfda_number`: `0.434` (required, string, nullable)
-+ `cfda_title`: `Flood Insurance` (required, string, nullable)
 + `awarding_agency` (required, Agency, fixed-type)
 + `funding_agency` (required, Agency, fixed-type, nullable)
 + `recipient` (required, Recipient, fixed-type)
 + `period_of_performance` (required, PeriodOfPerformanceAssistance, fixed-type)
 + `place_of_performance` (required, Location, fixed-type)
 + `executive_details` (required, Executive, fixed-type)
++ `cfda_info` (required, array[CFDAInfo], fixed-type)
+
+##CFDAInfo (object)
```

Comment:
This line should have a space after `##`

Reply:
good find, should be fixed now